examples/helpers/mod.rs

```diff
@@ -165,3 +135,12 @@ impl ExamplePage {
 }
 
 impl View for ExamplePage {}
+
+fn theme_selection_dropdown(cx: &mut Context) {
+    PickList::new(cx, ControlsData::theme_options, ControlsData::selected_theme, true)
+        .on_select(|cx, index| cx.emit(ControlsEvent::SetThemeMode(index)))
```

Comment:
Just a thought. Instead of emitting a `ControlsEvent` here and then handling that and translating it to an `EnvironmentEvent::SetThemeMode` event. Could do that here instead and cut out the middle man?

Reply:
Ah yes good point! Missed that part